lib/components/RoundButton/RoundButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeNaturaLight } from '../..';
```

Comment:
import straight from the themes file, not from index, since it will generate circular references.

RoundButton imports theme from index -> index exports RoundButton -> RoundButton imports from index...
It doesn't break at runtime, but this will generate warnings and will break babel parsing to ES5 when building the npm distribution

Reply:
@robertLichtnow got it, will fix this